doc/DESI_TARGET/targets.rst

```diff
@@ -39,65 +32,113 @@ FITS Header Units
 HDU0
 ----
 
-Empty header.
+Empty HDU.
 
 HDU1
 ----
 
-Targets.
+EXTNAME = TARGETS
+
+Target selection table
 
 Required Header Keywords
 ~~~~~~~~~~~~~~~~~~~~~~~~
 
-======== ======================================================== ==== ===================================
-KEY      Example Value                                            Type Comment
-======== ======================================================== ==== ===================================
-EXTNAME  TARGETS                                                  str  name of this binary table extension
-DEPNAM00 desitarget                                               str
-DEPVER00 0.1.0                                                    str  desitarget.__version__
-DEPNAM01 desitarget-git                                           str
-DEPVAL01 0.1.0                                                    str  git revision
-DEPNAM02 tractor-files                                            str
-DEPVER02 /project/projectdirs/cosmo/data/legacysurvey/dr1/tractor str  input directory
-======== ======================================================== ==== ===================================
+======== ==================== ==== ===================================
+KEY      Example Value        Type Comment
+======== ==================== ==== ===================================
+NAXIS1   243                  int  width of table in bytes
+NAXIS2   15050911             int  number of rows in table
+EXTNAME  TARGETS              str  name of this binary table extension
+DEPNAM00 desitarget           str
+DEPVER00 0.14.0               str
+DEPNAM01 desitarget-git       str
+DEPVER01 unknown              str
+DEPNAM02 sandboxcuts          str
+DEPVER02 F                    bool sandbox cuts were used or not (T/F)
+DEPNAM03 photcat              str
+DEPVER03 dr4                  str
+DEPNAM04 tractor-files        str
+DEPVER04 /data/dr4/sweep/4.0/ str  Full path to input catalogs
+DEPNAM05 qso-selection        str
+DEPVER05 randomforest         str  QSO selection algorithm
+DEPNAM06 HPXNSIDE             str
```

Comment:
Ooops...my mistake. I'll change the `HPXNSIDE` and `HPXNEST` KWs to be normal KWs in `desitarget.io`, at some point, and I'll update the data model myself once I've made that change.